udata/templates/macros/metadata.html

```diff
@@ -43,4 +43,9 @@
 <meta name="check-urls" content="{{ 'true' if config.CROQUEMORT else 'false' }}" />
 <meta name="check-urls-ignore" content="{{ config.CROQUEMORT_IGNORE|tojson|urlencode }}" />
 <meta name="territory-enabled" content="{{ 'true' if config.ACTIVATE_TERRITORIES else 'false' }}">
+
+{% if json_ld %}
+<script id="json_ld" type="application/ld+json">{{ json_ld|safe }}</script>
```

Comment:
For the same reason why there is this comment (https://github.com/opendatateam/udata/blob/dev/udata/frontend/helpers.py#L332).

If you do not put safe, the " are escaped and it does not generate a valid JSON.

Since the JSON is fully generated by the server, I think it can be considered as safe.